code/game/machinery/alarm.dm

```diff
@@ -39,7 +39,8 @@
 	anchored = 1
 	use_power = 1
 	idle_power_usage = 80
-	active_power_usage = 1000 //For heating/cooling rooms. 1000 joules equates to about 1 degree every 2 seconds for a single tile of air.
+	//active_power_usage = 1000 //For heating/cooling rooms. 1000 joules equates to about 1 degree every 2 seconds for a single tile of air.
+	active_power_usage = 15000 //Vorestation edit

```

Comment:
this seems unwise to supercharge _all_ the air alarms, and furthermore, this is not how you would make a vorestation edit in this scenario

you'd make a new DM file called alarm_vr.dm and change only that one var.

```
/obj/machinery/alarm
	active_power_usage = 15000
```

Reply:
Supercharging all the alarms would mean there wouldn't be a part two (and three, and four, and five...) that consists of selectively map-editing a hypothetical more powerful variant in, but okay. Fair enough on _vr.
Would it be better if I instead defined a few alarm subtypes with varying thermostat power ratings? Something like...
```
/obj/machinery/alarm/thermo2
    active_power_usage = 7500
/obj/machinery/alarm/thermo3
    active_power_usage = 15000
/obj/machinery/alarm/atrium
    active_power_usage = 100000

Reply:
I have to edit the map anyway. Why is this any better?

Reply:
You said that you'd rather not supercharge all air alarms everywhere. I'm trying to figure out what, if anything, I can do to make the idea acceptable.